tests/disk_check_test.py

```diff
@@ -26,7 +27,7 @@
         "workdir": "/tmp/tmpy",
         "mounts": "overlay_tmpx blahblah",
         "err": "/tmpx is not read-write|READ-ONLY: Mounted ['/tmpx'] to make Read-Write",
-        "cmds": ['mount -t overlay overlay_tmpx -o lowerdir=/tmpx,upperdir=/tmp/tmpx,workdir=/tmp/tmpy /tmpx']
+        "cmds": ['mount -t overlay overlay_tmpx -o lowerdir=/tmpx,upperdir=/tmp/tmpx/tmpx,workdir=/tmp/tmpy/tmpx /tmpx']
```

Comment:
>upperdir=/tmp/tmpx/tmpx [](http://example.com/codeflow?start=66&length=23)

Could you explain why you need test code change when you change implementation details? Could you make test code not depends on the implementation?

Reply:
This is a base change. We use explicit subdir under upper & work for each dir to mount.